llvm/test/CodeGen/AArch64/framelayout-sve-basepointer.mir

```diff
@@ -1,4 +1,5 @@
 # RUN: llc -mtriple=aarch64-none-linux-gnu -run-pass=prologepilog %s -o - | FileCheck %s
+# RUN: llc -mtriple=aarch64-none-linux-gnu -passes='prolog-epilog' %s -o - | FileCheck %s
```

Comment:
I don't think it's necessary to update every single -run-pass=prologepilog test to test both pass managers. We just need a sample here. For most tests it's excessive to test both